backend/routers/Pins.py

```
@@ -0,0 +1,45 @@
+from typing import List
+from sqlalchemy.orm import Session
+from database import get_db
+import models
+import schemas
+
+router = APIRouter(prefix="/pins", tags=["pins"])
+
+UPLOAD_DIR = "static/pins"
+
+@router.post("/upload")
+async def upload_pin_image(file: UploadFile = File(...)):
```

Ruff (0.14.8)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)